Drivers/IWDG_driver/Src/independent_watchdog.cpp

```diff
@@ -0,0 +1,27 @@
+#include "independent_watchdog.h"
+#include "main.h"
+
+/**
+ * @brief Borrow the configuration of an already existing watchdog and set it
+ * to the prescaler and Reload values.
+ *
+ */
+IndependentWatchdog::IndependentWatchdog(IWDG_HandleTypeDef *watchdog) {
+	this->watchdog = watchdog;
+	IWDG_InitTypeDef initDef = watchdog->Init;
+	this->iwdg_prescaler = initDef.Prescaler;
+	this->iwdg_reload = initDef.Reload;
```

Comment:
iwdg_prescaler and iwdf_reload dont exist anymore (and arent used) so probably remove lines 11,12 and 13 I think